README.md

```diff
@@ -46,7 +46,7 @@ This pipeline needs one of the following two inputs:
 
 1. A submission base directory path with a folder structure following [GRZ submission standard](https://github.com/BfArM-MVH/grz-cli?tab=readme-ov-file#introduction). You can also check [test datasets](https://www.cmm.in.tum.de/public/grz-example-submissions/).
 
-2. Use a csv samplesheet as input. This gives more flexibilty, as you don't need a GRZ submission directory.
+2. Use a csv samplesheet as input. This gives more flexibility providing optional starting points for the analysis either from fastq reads or BAM alignments, and you won`t need a GRZ submission directory.
```

Comment:
I would just say reads here instead of FASTQ, since PacBio reads come off the sequencer in [BAM format](https://pacbiofileformats.readthedocs.io/en/11.0/BAM.html).

Reply:
I updated t, yet you might need to rephrase it with long-read implementation.